### ISSUE TITLE
Adding window-information to time-sequence graphs

### ISSUE THREAD

**Issue by cpaasch** [nasa/multipath-tcp-tools#2]
Would be nice if there would be window-information in the time-sequence graphs genereate with mptcpplot (similar to what TCP-trace does).

**Comment by jishac** [comment]
Agreed, that would be a useful feature.  I will add it to the future enhancements list.
